parametric/conftest.py

```diff
@@ -411,7 +444,7 @@ def docker_run(
     r = subprocess.run(_cmd, stdout=log_file, stderr=log_file)
     if r.returncode != 0:
         pytest.fail(
-            "Could not start docker container %r with image %r, see the log file %r" % (name, image, log_file),
+            "Could not start docker container %r with image %r, see the log file %r" % (name, image, log_file.read()),
```

Comment:
```suggestion
            "Could not start docker container %r with image %r, see the log file %r" % (name, image, log_file),
```

I think this log message is meant to include the name of the log file, not its contents.

Reply:
For me, it returns something opaque, but I'm ok reverting this.